File: ejercicicos con jose cuellar/nuevos ejercicios diferentes/php/ejercicioplanilla2.php
<form action="#" method="post">
    <input type="number" name="nota1" plaseholder="ingrese la primera nota">
    <input type="number" name="nota2" plaseholder="ingrese la segunda nota">
    <input type="number" name="nota3" plaseholder="ingrese la tercera nota">
    <button type ="submit">calcular</button>
</form>

<?php
$nota1 = $_POST['nota1'];
$nota2 = $_POST['nota2'];
$nota3 = $_POST['nota3'];

$respuesta = ($nota1+$nota2+$nota3) / 3;

if ($respuesta < 69){
    echo "usted perdio el año con un priomedio de: ".$respuesta." muy malo jajajajaja";
}else {
    echo "usted gano el año con un promedio de ".$respuesta." felicitaciones";
}

File: ejercicicos con jose cuellar/nuevos ejercicios diferentes/payton/ejercicioplanilla2.py
print("nota de aprobacion")
nota1 = int(input("esctiba la primera nota"))
nota2 = int(input("escriba la segunda nota"))
nota3 = int(input("escriba la tercera nota"))
respuesta = (nota1+nota2+nota3)/3
if respuesta < 69:
    print("usted perdio el año ",respuesta," por malo jajajajaj")
if respuesta > 70:
    print("usted gano el año ",respuesta," felicidades")